LiteFarmOrg/LiteFarm
# Searching for a crop name in English on the crop catalogue when app is in another language, still returns English version of the name

To test: 
1. go to the profile page section: http://localhost:3000/profile
2. change the language to 'Spanish'.
3. go to http://localhost:3000/crop_catalogue
4. in the search type: tomato, check if you can find 'Tomate' in the results. If not, then it's working. 
5. in the search type: Tomate, check if you can find 'Tomate' in the results. If yes, then it's working only for the Spanish language.
